frontend/src/app/routes/auth-routing.module.ts

```diff
@@ -0,0 +1,29 @@
+import { RouterModule, Routes } from '@angular/router';
```

Comment:
If you have separate folder for routes, why app-routing.module is not in this folder?
Or maybe it would be better to store routing module with specific feature module, because routing is used there?

Reply:
Moved routing modules to their corresponding modules